addons/account/static/tests/x2many_buttons_tests.js

```diff
@@ -0,0 +1,155 @@
+/** @odoo-module **/
+import { click, editInput, getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+
+
+QUnit.module("Fields", (hooks) => {
+    let serverData;
+    let target;
+    const fromArch = `
+        <form>
+            <field name="display_name"/>
+            <field name="ref"/>
+            <field name="duplicated_ref_ids" widget="x2many_buttons"/>
+        </form>
+    `;
+
+    hooks.beforeEach(() => {
+        target = getFixture();
+        serverData = {
+            models: {
+                "account.move": {
+                    fields: {
+                        display_name: {
+                            "string": "Display Name",
+                            "type": "string"
+                        },
+                        duplicated_ref_ids: {
+                            "string": "Duplicated Bills",
+                            "type": "many2many",
+                            "relation": "account.move"
+                        },
+                        ref: {
+                            "string": "Bill Reference",
+                            "type": "char"
+                        }
+                    },
+                    records: [
+                        // for the sake of mocking data, we don't care about the consistency of duplicated refs across records
```

Comment:
the fuplicates are not unconsistent because if one record is a dupe of another the other should be a dupe of it which is not the case in the data i did it looks illogical but it is only for the sake of testing